src/pybind/mgr/dashboard/frontend/src/app/shared/components/cd-label/cd-label.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'cd-label',
+  templateUrl: './cd-label.component.html',
+  styleUrls: ['./cd-label.component.scss']
+})
+export class CdLabelComponent {
+  @Input() text?: string;
+  @Input() key?: string;
+  @Input() value?: string;
```

Comment:
No need to provide 2 sets of mutually exclusive inputs (text vs key-value). Why not just value = text and if key is defined then it behaves a key-value; otherwise as just value (text).

Reply:
I though it could be confusing when using the component, since the words 'key' and 'value' usually go as a pair. But I'll change it since it is cleaner this way. Thanks for all your comments and reviews Ernesto.